### ISSUE TITLE
Unsupported generated import in client_mapping.dart

### ISSUE THREAD

**Issue by mzdm** [epam-cross-platform-lab/swagger-dart-code-generator#167]
Swagger/Json file example: https://cloud.ibm.com/apidocs/visual-recognition/visual-recognition-v3.json
-> this has name `visual-recognition-visual-recognition-v3.swagger`

When running the generator, `client_mapping.dart` generates import:
```dart
import 'visual-recognition-visual-recognition-v3.swagger.dart';
```
Target of URI doesn't exist: 'visual-recognition-visual-recognition-v3.swagger.dart'.

However in other files the imports generates fine, `-` are replaced with `_`, e.g. `visual_recognition_visual_recognition_v3.swagger` file:

```dart
import 'package:json_annotation/json_annotation.dart';
import 'package:meta/meta.dart';
import 'package:chopper/chopper.dart';
import 'package:chopper/chopper.dart' as chopper;
import 'visual_recognition_visual_recognition_v3.enums.swagger.dart' as enums;
export 'visual_recognition_visual_recognition_v3.enums.swagger.dart';

part 'visual_recognition_visual_recognition_v3.swagger.chopper.dart';
part 'visual_recognition_visual_recognition_v3.swagger.g.dart';
```

<details>
<summary>deps</summary>

```
dependencies:
  flutter:
    sdk: flutter
  chopper: ^4.0.0

dev_dependencies:
  flutter_test:
    sdk: flutter
  build_runner: ^1.12.2
  json_annotation: 4.0.1
  json_serializable: 4.1.2
  swagger_dart_code_generator: ^2.0.0-nullsafety.7+2
  chopper_generator: ^4.0.0-nullsafety.1
```

</details>

**Comment by Vovanella95** [comment]
Please check 2.0.0-nullsafety.8

**Comment by mzdm** [comment]
Thanks, fixed with `2.0.0-nullsafety.8`!
